### ISSUE TITLE
Colima daemon does not find volumes with --mount-inotify and --runtime containerd

### ISSUE THREAD

**Issue by flokain** [abiosoft/colima#1153]
There is a bug with nerdctl where nerdctl inspect <container1> <container2>.... returns invalid json in form of [{container1}][{container2}]... instead of [{container1},{container2}] its  addressed in this bugissue on nerdctl https://github.com/containerd/nerdctl/issues/3006

By the way this requires nerdctl 2.0 which is not shipped with k3s versions at the moment and the  nerdctl 1.7.6. version they use doesnt have a backported fix for https://github.com/containerd/nerdctl/pull/2939

```
colima status -p devenv
INFO[0000] colima [profile=devenv] is running using macOS Virtualization.Framework 
INFO[0000] arch: aarch64                                
INFO[0000] runtime: containerd                          
INFO[0000] mountType: virtiofs                          
INFO[0000] address: 192.168.106.3                       
INFO[0000] kubernetes: enabled
```

I have verified that with those two fixes the deamon works as expected.

**Comment by abiosoft** [comment]
Yeah, I am avoiding updating nerdctl to v2.0 before it's stable release.
